docs/reference/deploy/config/master-config-reference.rst

```diff
@@ -500,10 +500,10 @@ created by the launcher. The default is ``ALL``. The specification ``!root`` is
 appended to this list to prevent privilege elevation. See the ``sudoers(5)`` definition of
 ``Runas_List`` for the full syntax of this value. See :ref:`sudo_configuration` for details.
 
-``singularity_image_root``
---------------------------
+``apptainer_image_root`` or ``singularity_image_root``
```

Comment:
To add this you need to bump the launcher version we reference /setup-cluster/deploy-cluster/slurm/overview

Reply:
Fixed.